cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/cruisecontrol/CruiseControlEndpoints.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.assembly.cruisecontrol;
+
+enum CruiseControlEndpoints {
```

Comment:
Just a point to note about this pattern: an `enum` like this works fine when all your paths are static (as here), but not when the path contains some parameters, e.g. `/foo/${name}`.